render/wgpu/src/backend.rs

```diff
@@ -150,11 +147,7 @@ impl WgpuRenderBackend<crate::target::TextureTarget> {
 }
 
 impl<T: RenderTarget> WgpuRenderBackend<T> {
-    pub fn new(
-        descriptors: Arc<Descriptors>,
-        target: T,
-        preferred_sample_count: u32,
-    ) -> Result<Self, Error> {
+    pub fn new(descriptors: Arc<Descriptors>, target: T) -> Result<Self, Error> {
```

Comment:
Could this take in the quality as an argument? That would allow us to avoid re-creating the surface immediately when the command-line `quality` argument is something other than `StageQuality::Low`

Reply:
We have to recreate it (and the actual wgpu Surface it's for) immediately for the size of the swf and other player configs that may affect it. I think there's a better way to do the whole bootstrapping process overall, probably by passing in ` render builder` to the PlayerBuilder, but I'm not sure.